release/images.sh

```diff
@@ -36,6 +36,30 @@ sign_checksum() {
     fi
 }
 
+finalize_image() {
+    local name=${1}
+
+    local finalize_script=$(find "${TEMPLATES_PATH}" -name "${name}-finalize.*" -print -quit)
+    if [[ -n "${finalize_script}" ]]; then
+        local script_log="${LOG_DIR}/${name}-finalize"
+        "${finalize_script}" "${WORK_DIR}/${name}.img" > "${script_log}.log" 2> >(tee "${script_log}_err.log")
+
+        if [[ -s "${script_log}_err.log" ]]; then
+            log "Image '${name}'" "NOT OK!"
+            return 1
+        fi
```

Comment:
You can combine all of this with something like:
```
if ! LOG_METHOD=1 run_and_log "${finalize_script}" "${WORK_DIR}/${name}.img"; then
    log "Finalize step for ${name} image failed" "NOT OK!"
    return 1
fi
```

Reply:
Updated that